river/XdgToplevel.zig

```diff
@@ -276,7 +276,10 @@ fn handleCommit(listener: *wl.Listener(*wlr.Surface), surface: *wlr.Surface) voi
             } else {
                 const self_tags_changed = view.pending.tags != view.current.tags;
                 view.current = view.pending;
-                if (self_tags_changed) view.output.sendViewTags();
+                if (self_tags_changed) {
+                    view.output.sendViewTags();
+                    view.output.sendUrgentTags();
```

Comment:
This logic also needs to be fixed in the same way as the logic in `Root.commitTransaction()` described above.